lib/Config.ts

```diff
@@ -115,24 +134,27 @@ class Config {
     if (args) {
       deepMerge(this.config, args);
     }
-
     return this.config;
   }
 
   private parseIniConfig = (filename: string, mergeTarget: any, configType: string) => {
     if (fs.existsSync(filename)) {
       try {
-        const config = ini.parse(fs.readFileSync(filename, 'utf-8'));
-        const { rpcuser, rpcpass, listen } = config['Application Options'];
-
-        rpcuser ? mergeTarget.user = rpcuser : undefined;
-        rpcpass ? mergeTarget.password = rpcpass : undefined;
-
-        if (listen) {
-          const split = listen.split(':');
-          mergeTarget.host = split[0];
-          mergeTarget.port = split[1];
+        const config = ini.parse(fs.readFileSync(filename, 'utf-8'))['Application Options'];
+
+        switch (configType) {
+          case 'LND':
```

Comment:
Because we have just two cases right now: what about a `isLndConfig` boolean instead of `configType`?